src/Lykke.Job.OperationsCache/Services/InMemoryCache/RedisStorage.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Common;
+using Lykke.Job.OperationsCache.Models;
+using Microsoft.Extensions.Caching.Distributed;
+
+namespace Lykke.Job.OperationsCache.Services.InMemoryCache
+{
+    public class RedisStorage : IStorage
+    {
+        private readonly IDistributedCache _redisCache;
+        
+        public RedisStorage(IDistributedCache redisCache)
+        {
+            _redisCache = redisCache;            
+        }
+
+        public async Task<IEnumerable<HistoryEntry>> Get(string clientId)
+        {
+            string value = await _redisCache.GetStringAsync(GetCacheKey(clientId));
+
+            return value?.DeserializeJson<CacheModel>().Records.Values.Select(v => v);
```

Comment:
I found NetJSON is much faster for serialization/deserialization. Maybe it would be better to use it here, with huge amount of data.

Reply:
maybe :) we will optimize it in case of performance problems.